.jenkins/pytorch/test.sh

```diff
@@ -438,6 +448,7 @@ elif [[ "${BUILD_ENVIRONMENT}" == *-test1 || "${JOB_BASE_NAME}" == *-test1 ]]; t
   test_python_shard1
   test_aten
 elif [[ "${BUILD_ENVIRONMENT}" == *-test2 || "${JOB_BASE_NAME}" == *-test2 ]]; then
+  test_python_gloo_with_tls
```

Comment:
put it at first place for testing purpose, will move it to the end after resolve all issues

Reply:
Please make sure to limit it to just few configs.

Reply:
To elaborate on my previous comment, invoke this function if BUILD_ENVIRONEMNT equals to `pytorch_linux_xenial_py3_6_gcc7_test`

Reply:
addressed